lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -4264,7 +4264,7 @@ void initMIDIPrimitives();
 	initMIDIPrimitives();
 #endif
 
-#if !defined(_WIN32) && !defined(SC_IPHONE) && !defined(__OpenBSD__) && !defined(__NetBSD__) && !defined(__APPLE__)
+#if defined __linux__
```

Comment:
@danstowell the LID file is only included if LINUX is defined in Cmake, which is only defined if CMAKE_SYSTEM_NAME is "Linux", so I think this is safe. Do you see any issues arising from that? `__linux__` is after all only defined on the Linux kernel.